hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -861,28 +923,35 @@
 
     // ext is to create port in the top.
     'external': {
-       'clkmgr_aon.clk_main'           : 'clk_main',  // clock inputs
+        'ast.edn'                      : '',
+        'ast.lc_dft_en'                : '',
+        'ast.ram_1p_cfg'               : 'ram_1p_cfg',
+        'ast.ram_2p_cfg'               : 'ram_2p_cfg',
+        'ast.rom_cfg'                  : 'rom_cfg',
```

Comment:
I think the port name having `ast_` prefix shows more meaningful name. I thought at a glance, the ram_1p_cfg is for entire 1 port rams inside SoC.

Reply:
wait actually, the ram_1p_cfg is meant for all 1 port rams inside SoC. Similarly ram_2p_cfg is meant for all 2p rams. 

The main reason i didn't prefix them with 'ast' is because just in case in the future these values don't come from ast.